modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContextTest.java

```diff
@@ -362,7 +362,7 @@ public void testGetDDMFormContext() throws Exception {
 
 		ddmForm.addDDMFormField(ddmFormField);
 
-		Mockito.doReturn(
+		DDMForm ddmForm1 = Mockito.doReturn(
```

Comment:
@Marina-Gouveia why do we need to assign to this variable? It is never used

Reply:
Hi @carolmariaabb, I made the necessary adjustment, but it ended up going through inattention.